install/kubernetes/helm/Istio/requirements.yaml

```diff
@@ -0,0 +1,13 @@
+dependencies:
+  - name: pilot
+    version: 0.1.0-1
+    condition: Istio.pilot_enabled
+  - name: auth
+    version: 0.1.0-1
+    condition: Istio.auth_enabled
+  - name: mixer
```

Comment:
mixer will always be enabled, hence not require a condition variable.

Reply:
ditto, forcing something to be always on with any serious technical argument, is not right approach

Reply:
Mixer is optional.

Reply:
Put another way, mixer is two components : telemetry and policy enforcement. So it would need two conditions.

Reply:
After reading the steering committee thread on this topic, these should be deployable individually.  I question how valuable that would be in the context of a Helm chart which only targets Kubernetes whereas the steering committee debate related to platforms other than Kubernetes.

Cheers
-steve